src/scenes/OBG/scenes/ObgContent/scenes/ObgList/components/AdList/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import ListItem from './components/ListItem';
+import styles from './styles.scss';
+
+const AdList = ({ items }) => {
+  const keys = items.keySeq();
+
+  return (<div className={styles.wrapper}>
+    {keys.map(key => (<ListItem
+      key={key}
+      item={items.get(key)}
+    />)
+    )}
+  </div>);
+};
+
```

Comment:
The above styling can be improved because it's pretty hard to read the JSX. 

```
return (
  <div className={styles.wrapper}>
    {keys.map(key => <ListItem key={key} item={items.get(key)} />)}
  </div>
);
```